packages/palette/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line no-undef
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:@typescript-eslint/recommended",
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 12,
+    sourceType: "module",
+  },
+  plugins: ["react", "react-hooks", "@typescript-eslint"],
+  rules: {
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn",
+  },
+}
```

Comment:
How was this not already in? Ah, we were using [ts-lint](https://github.com/artsy/palette/blob/master/packages/palette/tslint.json). Can we remove refs, etc?

Reply:
👍